error_registration.py

```diff
@@ -53,7 +53,7 @@ def getFitnessErrorMsgs(schedule_by_dp):
     err_msgs.append("Too many PE's, maximum is two.")
   if num_fitness < 2:
     err_msgs.append("At least two PE or PE alternatives required.")
-  if num_fitness > 3:
+  if num_fitness > 2:
     err_msgs.append("Too many PE or PE alternatives, maximum is three.")
```

Comment:
Shouldn't we also change the error message for this on the following line from "maximum is three" to "maximum is two"?

Reply:
Good catch, thanks.
error_registration.py has been deleted and the functionality is moved to reports.py. The error message looks correct there.

Reply:
I'm looking at reports.py and see a line that doesn't make sense to me. Is this right?
```if (num_PE < 1 and num_Dance < 2):```
I would think that both should be `< 1`, but maybe I'm misunderstanding something there.

Reply:
This is fine.
Each student is required to take at least one PE. The only exception is if a two-block Dance class is being offered, it could be taken as a replacement for PE.
So this test says: if the student is not taking PE and is not taking the two-block Dance, then give an error.
(The two-block Dance class hasn't been offered for years, but let's leave this code in case the class is ever offered again in the future.)